a/Assembler 8048 videopac.asm

```diff
@@ -44,7 +44,7 @@ nextchar:
 loop:
 	jmp 	loop                            ; just wait and do nothing
 
-; 'HELLO WORLD' (ascii not supported by assembler)
+; 'Hello World' (ascii not supported by assembler)
```

Comment:
This is actually just a comment - the string that is printed also needs to be encoded.

Reply:
Sorry, I don’t know how to do that.

Reply:
@Richienb do you?

Reply:
Since ASCII is not supported by assembler, there is no way to do it so this comment should just be left as is.

Reply:
Ok

Reply:
done